INTITA/Modules/ForgotPassword/ForgotPasswordViewController.swift

```diff
@@ -13,67 +13,151 @@ class ForgotPasswordViewController: UIViewController, Storyboarded {
+        view.layoutSubviews()
+        forgotTableView.endUpdates()
+        
+    }
+    
+    func registerCells() {
+        let logoCell = UINib(nibName: "LogoForgotTableViewCell", bundle: nil)
+        forgotTableView.register(logoCell, forCellReuseIdentifier: "LogoForgotTableViewCell")
+        
+        let passCell = UINib(nibName: "PassRecoveryTableViewCell", bundle: nil)
+        forgotTableView.register(passCell, forCellReuseIdentifier: "PassRecoveryTableViewCell")
+        
+        let textCell = UINib(nibName: "ExplanationTableViewCell", bundle: nil)
+        forgotTableView.register(textCell, forCellReuseIdentifier: "ExplanationTableViewCell")
         
+        let emailCell = UINib(nibName: "EmailTableViewCell", bundle: nil)
+        forgotTableView.register(emailCell, forCellReuseIdentifier: "EmailTableViewCell")
+        
+        let registerButtonCell = UINib(nibName: "SendButtonTableViewCell", bundle: nil)
+        forgotTableView.register(registerButtonCell, forCellReuseIdentifier: "SendButtonTableViewCell")
     }
     
-    func setupSendButton() {
+}
+
+extension ForgotPasswordViewController: UITableViewDelegate {
         
-        let titleLocalized = "send".localized
-        sendButton.layer.cornerRadius = 8
-        sendButton.setTitle(titleLocalized, for: .normal)
+    }
+
+extension ForgotPasswordViewController: UITableViewDataSource {
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 9
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+       
+        switch indexPath.row {
+        case 1:
+            guard let logoCell = tableView.dequeueReusableCell(withIdentifier: "LogoForgotTableViewCell") as? LogoForgotTableViewCell else { return UITableViewCell() }
+            return logoCell
+        case 3:
+            guard let passRecoveryCell = tableView.dequeueReusableCell(withIdentifier: "PassRecoveryTableViewCell") as? PassRecoveryTableViewCell else { return UITableViewCell() }
+            return passRecoveryCell
+        case 5:
+            guard let explanationCell = tableView.dequeueReusableCell(withIdentifier: "ExplanationTableViewCell") as? ExplanationTableViewCell else { return UITableViewCell() }
+            return explanationCell
+        case 6:
+            guard let emailCell = tableView.dequeueReusableCell(withIdentifier: "EmailTableViewCell") as? EmailTableViewCell else { return UITableViewCell() }
+            emailCell.textField.placeholder = "inputEmail".localized
+            emailCell.textField.textContentType = .emailAddress
+            return emailCell
+        case 8:
+            guard let buttonCell = tableView.dequeueReusableCell(withIdentifier: "SendButtonTableViewCell") as? SendButtonTableViewCell else { return UITableViewCell() }
+            buttonCell.delegate = self
+            return buttonCell
+        default:
+            return UITableViewCell()
```

Comment:
Please refactor it like this: declare let cell before the switch, delete all returns and assign a value to a cell at the end of each case, in default case return cell ?? UITableViewCell()